src/main/java/com/toofifty/easyblastfurnace/methods/GoldHybridMethod.java

```diff
@@ -49,11 +49,6 @@ public MethodStep next(BlastFurnaceState state)
         // continue doing gold bars until enough coal has been deposited
         // then do one trip of metal bars
 
-        if (state.getInventory().has(ItemID.GOLD_ORE) &&
-            !state.getEquipment().equipped(ItemID.GOLDSMITH_GAUNTLETS, ItemID.SMITHING_CAPE, ItemID.SMITHING_CAPET)) {
-            return equipGoldsmithGauntlets;
-        }
```

Comment:
Correct me if I'm wrong, but I think this one is needed to make sure the player has the gold gauntlets equipped before they drop them in the hopper (if they have ice gloves currently equipped)

Reply:
I'll double check, I tested it but maybe I only tested with a smithing cape and not with the goldsmith gauntlets

Reply:
Yep you do need that, that's my bad 😬 I really need to stop testing with the smithing cape